### ISSUE TITLE
Testing `pack-<model>-src`

### ISSUE THREAD

**Issue by Expander** [FlexibleSUSY/FlexibleSUSY#210]
Can we add a test to check that `pack-<model>-src` works?
It could look like this (after the meta code has run):
~~~
make pack-SM-src  # create tarball with model source code
rm -rf models/SM/ # remove model source code
tar xf SM.tar.gz  # extract tarball
./configure --with-models=SM
make -j2
~~~

**Comment by wkotlarski** [comment]
Just make it 100% clear: you don't mean a test like something in `test` directory but an extra step in github action?

**Comment by Expander** [comment]
Yes, I mean a test in form of a github action. Sorry for being not precise.

**Comment by wkotlarski** [comment]
Ok. So you'd have to add  `pack-MODEL-src` options to makefile as it is not there right now, right?

**Comment by Expander** [comment]
It is already there. :) For each model `m` there is a corresponding make target called `pack-m-src`.

I'd propose to perform this test only for one simple model that is enabled in the tests, for example the `SM`. (See my code snippet above.)

**Comment by wkotlarski** [comment]
Implemented in PR #212. If test pass I'll merge it into development.

**Comment by Expander** [comment]
Cool, many thanks!

**Comment by wkotlarski** [comment]
> Can we add a test to check that `pack-<model>-src` works?
> It could look like this (after the meta code has run):
> 
> ```
> make pack-SM-src  # create tarball with model source code
> rm -rf models/SM/ # remove model source code
> tar xf SM.tar.gz  # extract tarball
> ./configure --with-models=SM
> make -j2
> ```

@Expander, shouldn't configure also have `--disable-meta`?

**Comment by Expander** [comment]
Yes! Sorry, I missed that.

**Comment by wkotlarski** [comment]
Looks like `test/test_SM_cxxdiagrams.meta` wants `/FlexibleSUSY/models/SM/FlexibleSUSY.m` which is not present in SM.tar.gz. Maybe `--disable-meta` is not passed to `test/module.mk`?

**Comment by Expander** [comment]
Yes, this sounds like a bug. I'll fix it in the development branch.

What exactly are you executing which leads to `test/test_SM_cxxdiagrams.meta` being build?

**Comment by wkotlarski** [comment]
Please see [here](https://github.com/FlexibleSUSY/FlexibleSUSY/runs/674506193?check_suite_focus=true). First we 
```
Building test/test_SM_cxxdiagrams.cpp
```
which gives
```
Error: model file not found: /FlexibleSUSY/models/SM/FlexibleSUSY.m
```
This somehow does not stop the tests. They go through and only finally fail during
```
make: *** No rule to make target 'test/test_SM_cxxdiagrams.cpp', needed by 'test/test_SM_cxxdiagrams.o'.  Stop.
make: *** Waiting for unfinished jobs....
```

**Comment by Expander** [comment]
Many thanks for the hint! I've tried to fix it in commit cade014c7 .
